src/main/target/OMNIBUSF4/target.c

```diff
@@ -83,8 +83,10 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
 #endif
     { TIM8,  IO_TAG(PC6),  TIM_Channel_1, TIM8_CC_IRQn,        0, IOCFG_AF_PP_PD, GPIO_AF_TIM8  }, // Connected: UART6 TX, not used as PWM, definition inherited from REVO target
     { TIM8,  IO_TAG(PC7),  TIM_Channel_2, TIM8_CC_IRQn,        0, IOCFG_AF_PP_PD, GPIO_AF_TIM8  }, // Connected: UART6 RX, not used as PWM, definition inherited from REVO target
+#if !defined(OMNIBUSF4V3)
```

Comment:
@jflyper what would happen if I would still leave those PWM in definition for V3? Or rather, what was the reason to remove even if they are not connected to any pads?